src/test/controllers/test-controller-spec.js

```diff
@@ -9,7 +9,8 @@ describe('TestController', function() {
     inject(function($injector) {
       $rootScope = $injector.get('$rootScope');
       $scope = $rootScope.$new();
-      controller = $injector.get('$controller')('TestController', {$scope: $scope});
+      controller = $injector.get('$controller')('TestController',
+      {$scope: $scope});
```

Comment:
add one step indentation. When beaking a line of code, indent the next of break by one step so as to help know that it belongs to the previous line.